README.md

```diff
@@ -15,3 +15,52 @@ This repository houses the open-source components of Mattermost Private Cloud. O
 ## Get Involved
 
 - [Join the discussion on Private Cloud](https://community.mattermost.com/core/channels/cloud)
+
+## Developing
+
+### Environment Setup
+
+1. Install [Go](https://golang.org/doc/install)
+2. Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) version v0.11.14
+3. Install [kops](https://github.com/kubernetes/kops/blob/master/docs/install.md) version 1.11.1
+4. Install [Helm](https://helm.sh/docs/using_helm/)
+5. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+6. Generate an AWS Access and Secret key pair, then export them in your bash profile:
+  ```
+  export AWS_ACCESS_KEY_ID=YOURACCESSKEYID
+  export AWS_SECRET_ACCESS_KEY=YOURSECRETACCESSKEY
+  ```
+7. Create an S3 bucket to store the kops state
+8. Clone this repository into your GOPATH
```

Comment:
nit: we are using go modules so it technically could be outside of the GOPATH. We should note that you need to enable go modules when building if it is in your GOPATH.